EmptyNode/azuredeploy.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the existing HDInsight cluster"
+            }
+        },
+        "galleryPackageIdentity": {
+            "type": "string",
+            "metadata": {
+                "description" : "Any non-empty value is accepted. After the package is published, this parameter will be used to link the application install to the details about the application in the marketplace."
+            },
+            "defaultValue": "EmptyNode"
+        },
+        "edgeNodeSize": {
+            "type": "string",
+            "metadata": {
+                "description": "Size of the edge node that hosts the application"
+            },
+            "defaultValue": "Standard_D3",
+            "allowedValues": [
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4"
+            ]
+        },
+        "edgeNodePrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Edgenode prefix"
+            },
+            "defaultValue": "new"
+        },
+        "instanceCount": {
```

Comment:
We have to set this as a variable until we see what the cost is of changing this in our workflows. There is a strict validation that you can only have one instance. We'll have to dig in further to determine whether that was scoping for testing or if there's other implications in our code that could fail if instanceCount > 1